Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -1149,6 +1149,10 @@ private WeenieError CheckWieldRequirement(WorldObject item)
             if (!PropertyManager.GetBool("use_wield_requirements").Item)
                 return WeenieError.None;
 
+            var allowedWielder = item.GetProperty(PropertyInstanceId.AllowedWielder);
+            if (allowedWielder != null && (allowedWielder != Guid.Full))
+                return WeenieError.YouDoNotOwnThatItem; // Unsure of the exact message
```

Comment:
I know this doesn't fit the pattern but it might be WeenieErrorWithString.Only_MayUseItemMagic

Reply:
That message seems to fit more with the AllowedActivator, rather than the AllowedWielder.  To me, with AllowedActivator present, one could at least equip the item.  They would just be prevented from using the spell or what have you upon it.

Reply:
you're probably right, my only thought was maybe the client interpreted it differently depending what it knows about that object